package.json

```diff
@@ -22,6 +22,7 @@
     "date-fns": "^2.29.3",
     "eslint": "8.38.0",
     "eslint-config-next": "13.3.0",
+    "feed": "^4.2.2",
```

Comment:
This looks good, but let's not add a new `^` dependency. These can be used to exploit the code base. We should pin all the other dependencies in another PR too.